src/test/resources/blockReturn.txt

```diff
@@ -1,6 +1,4 @@
 '- javadoc
-   |- [58]
-   |  '- [58]
```

Comment:
@rnveach @mahfouz72 this was an example of ANTLR making imaginary tokens up to overcome parsing issues